middlewares/middlewares.go

```diff
@@ -102,11 +105,45 @@ func AccessControl() negroni.Handler {
 	})
 }
 
+// ErrNoPEMKey if jwt.key should contain PEM data, but no valid blocks were found
+var ErrNoPEMKey = errors.New("no RSA/EC/PUBLIC PEM data found")
+
+func parsePEMKey(data []byte) (key interface{}, err error) {
+	var block *pem.Block
+	for len(data) > 0 {
+		block, data = pem.Decode(data)
+		if block == nil {
+			return nil, ErrNoPEMKey
+		}
+		switch block.Type {
+		case "RSA PRIVATE KEY":
+			return x509.ParsePKCS1PrivateKey(block.Bytes)
+		case "EC PRIVATE KEY":
+			return x509.ParseECPrivateKey(block.Bytes)
+		case "PUBLIC KEY":
+			return x509.ParsePKIXPublicKey(block.Bytes)
+		}
+	}
+	return
+}
+
 // JwtMiddleware check if actual request have JWT
 func JwtMiddleware(key string, algo string) negroni.Handler {
```

Comment:
We changed the jwt library (for one that was not archived), you can update your contribution with the `main` branch

**sorry**
ref: #635